index.html

```diff
@@ -8,7 +8,25 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <nav></nav>
-    <div class="main-container"></div>
+    <nav class="flex-div">
+        <div class="nav-left">
+            <h3 class="logo">My Logo</h3>
+        </div>
+        <div class="nav-right">
+            <img src="src/img/Union.png" alt="des-menu" class="menu">
+        </div>
+    </nav>
+    <section class="main-container">
+        <div class="greeting-text">
+            <h2 class="name">I'm Jhonatan.<br>Glad to see you!</h2>
+            <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="connect">
+            <p>LET'S CONNECT</p>
+        </div>
+        <div class="media-icon">
+            <img src="src/img/social media icons.png" alt="">
+        </div>
```

Comment:
- [ ] Kindly consider replacing this div with a  `ul` tag. then you must place each icon image in an `a` tag. that `a` tag will be inside each `li`.  the structure should look like this. by following it the visitors can easily reach out to you using these social icon links.
<ul>
  <li>
    <a>
      <img> 
    </a>
  </li>
  ..........
</ul>